### DISCUSSION TITLE
Win 10: Ethernet not connecting but WIFI works.

### DISCUSSION THREAD

**Discussion by RCJacH** [apohl79/audiogridder#323] [Help]
I have a laptop and a headless desktop, both running windows 10 and are connected through an ethernet cable. Both computers are in the same local WIFI as well. I'm trying to connect from desktop (controlled through rdp) back to the laptop.
I can connect using the WIFI address of the laptop, every time, with unbearable cracks. However I cannot connect them with the ethernet address, or by server name, which did appear automatically...it just won't connect.
Is there some thing I forgot to do, or is there a specific setup procedure that I need to go through to have them connect through ethernet?

**Comment by CaBuffalo** [comment]
Are you using a router? If no, you need a crossover cable to connect your
two PCs directly.

On Sun, Jan 24, 2021 at 11:31 AM RCJacH <notifications@github.com> wrote:

> I have a laptop and a headless desktop, both running windows 10 and are
> connected through an ethernet cable. Both computers are in the same local
> WIFI as well. I'm trying to connect from desktop (controlled through rdp)
> back to the laptop.
> I can connect using the WIFI address of the laptop, every time, with
> unbearable cracks. However I cannot connect them with the ethernet address,
> or by server name, which did appear automatically...it just won't connect.
> Is there some thing I forgot to do, or is there a specific setup procedure
> that I need to go through to have them connect through ethernet?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/323>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BJZQTBPR54RXULK23S3RYPXANCNFSM4WQYXU7Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by rockumk** [comment]
If you turn wifi off does the Ethernet start working? I requested the ability to be able to select a Network Interface in the preferences for this reason. I have two Ethernet networks and unfortunately it will work with either, but when both are on I don't get to choose and the one I don't want gets the traffic.

**Comment by CaBuffalo** [comment]
Yes, once wifi is off and ethernet starts working.

But you need to make sure both of your systems are in the same subnet.
Better use a router if you are not sure how to set up IP manually.

On Thu, Jan 28, 2021 at 9:50 AM rockumk <notifications@github.com> wrote:

> If you turn wifi off does the Ethernet start working? I requested the
> ability to be able to select a Network Interface in the preferences for
> this reason. I have two Ethernet networks and unfortunately it will work
> with either, but when both are on I don't get to choose and the one I don't
> want gets the traffic.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/323#discussioncomment-318503>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GOSNXK635SGPFVXSDS4GPVPANCNFSM4WQYXU7Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by rockumk** [reply]
The connection is fine. Audiogridder is taking the primary route. Other routes may be available and even preferable, but audiogridder ignores them. There is no way to select the NIC/route/network you wish to use.

Other software like Copperlan (creates a MIDI web over a network), NDI for video and audio over IP, Dante, and others allow you to select the NIC/Network you wish to use.

**Reply by apohl79** [reply]
AG server is actually binding to all interfaces. If you have the same networks available at your DAW machine, you can't choose the one you wish to use by using the auto discovery name. But you can manually add the endpoint you want to connect to, by adding a server and using the IP of the appropriate interface.

**Reply by rockumk** [reply]
My guess is that this is an issue when two (or more) network interfaces have access to the same subnet IP address pool. Binding occurs on all network interfaces so you can reach addresses on either interface and you can choose between the server's interfaces by IP, but you can't assure which client interface will be used to get there.

**Comment by CaBuffalo** [comment]
OIC. Unfortunately, AudioGridder doesn't have that function. I use ethernet
by disabling the Wifi on my systems.

On Thu, Jan 28, 2021 at 11:46 AM rockumk <notifications@github.com> wrote:

> The connection is fine. Audiogridder is taking the primary route. Other
> routes may be available and even preferable, but audiogridder ignores them.
> There is no way to select the NIC/route/network you wish to use.
>
> Other software like Copperlan (creates a MIDI web over a network), NDI for
> video and audio over IP, Dante, and others allow you to select the
> NIC/Network you wish to use.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/323#discussioncomment-318776>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6E4RCHZXX6ITEKZTILS4G5JJANCNFSM4WQYXU7Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
that’s true

**Comment by wpostma** [comment]
Turning off wifi is preferable not only because it will force AudioGridder to use the wired network but also because it will decrease your overall system latency on both client PC and server PC, and result in a net increase of  your DSP available power on both machines.
WIFI is one of the largest contributers to global system latency on both Windows and Mac.
